lntest/itest/lnd_multi-hop_htlc_local_chain_claim_test.go

```diff
@@ -420,3 +427,44 @@ func waitForInvoiceAccepted(t *harnessTest, node *lntest.HarnessNode,
 		}
 	}
 }
+
+// checkPaymentSucceeded asserts that the given node list a payment with the
+// given preimage as succeeded.
+func checkPaymentSucceeded(ctxt context.Context, node *lntest.HarnessNode,
```

Comment:
Seems better to pass in the payment status here instead to make it more general?

Reply:
Done: https://github.com/lightningnetwork/lnd/pull/3194/commits/b9a4971a964538b7eb9b97404f7648499b630727